plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -113,6 +113,62 @@ protected DistributedQueryRunner createQueryRunner()
         }
     }
 
+    @Test
+    public void testCheckpointFileOperations()
```

Comment:
can you add the test in a preparatory commit so that the "gains" are easier visible where you add caching?

Reply:
Fair call 👍